packages/fields/src/fields/comment-status/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import type { Field } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePost } from '../../types';
+
+const commentStatusField: Field< BasePost > = {
```

Comment:
Moved verbatim from `packages/edit-site/src/components/post-fields/index.js`. Only added the TypeScript bits.